src/app/components/iterations-panel/iterations-panel.component.html

```diff
@@ -1,57 +1,74 @@
 <!--Iteration Start-->
-<div class="iterations-container">
-  <div class="iteration-wrapper">
+<div class="f8-itr">
+  <div class="f8-itr-wrapper">
+    <!-- display the active iterations first -->
+
   <!-- Active iteration -->
-    <div class="iterations">
-      <div class="iteration-header pointer active-iteration-header">
-        <h3>
+    <div class="f8-itr__panel">
+      <div class="f8-itr__panel-hdr pointer f8-itr__panel-hdr--active">
+        <h3 class="padding-0 margin-0">
           Active Iteration
         </h3>
       </div>
-      <ul class="iteration-expanded-div">
-        <li class="active iteration-container" *ngFor="let iteration of activeIterations" [routerLinkActive]="'selected-iteration'"
+      <ul class="f8-itr__List--expanded">
+        <li class="active f8-itr__container"
+          *ngFor="let iteration of activeIterations"
+          [routerLinkActive]="'io-itr--selected'"
           [routerLink]="[]" [attr.data-id]="iteration.id"
```

Comment:
The selected active iteration does not get highlighted. The iteration gets highlighted in the tree view also.
Check what happens when you click on Sprint 137 in https://openshift.io/openshiftio/openshiftio/plan
![iteration](https://user-images.githubusercontent.com/1892722/30213824-d2516456-9499-11e7-9700-302b886519a8.png)